pkg/microservice/aslan/core/environment/handler/policy.yaml

```diff
@@ -6,33 +6,39 @@ rules:
     alias: "查看集成环境"
     description: ""
     rules:
+      - method: GET
+        endpoint: "/api/aslan/environment/environments/?*/services/down"
```

Comment:
It looks like this endpoint does not exist.

Reply:
fixed down->?*